test/Tests/src/Data/Time/Time_Of_Day_Spec.enso

```diff
@@ -117,8 +117,6 @@ specWith name create_new_time parse_time =
             time_1>time_2 . should_be_true
             time_1<time_2 . should_be_false
 
-            time_2.compare_to (Date_Time.new 1999 1 1 4 0 10) . should_fail_with Type_Error.Error
-
```

Comment:
can we have a test of what we get when comparing time with a date?

Reply:
Sure. Added. This was probably just a leftover.